builder/dockerfile/dispatchers.go

```diff
@@ -157,6 +158,7 @@ func add(req dispatchRequest) error {
 	if err != nil {
 		return err
 	}
+	copyInstruction.flChown = flChown.Value
```

Comment:
I think we should add it to both (even though we don't want to make `ADD` more magical); if we don't add it to `ADD`, people using it for adding remote files won't be helped by this PR.